tvx/lotus/driver.go

```diff
@@ -51,15 +51,12 @@ func (d *Driver) ExecuteMessage(msg *types.Message, preroot cid.Cid, bs blocksto
 		Syscalls:       mkFakedSigSyscalls(vm.Syscalls(ffiwrapper.ProofVerifier)),
 		CircSupplyCalc: nil,
 		BaseFee:        BaseFee,
+		NtwkVersion: func(context.Context, abi.ChainEpoch) network.Version {
+			// TODO: Network upgrade.
+			return network.Version0
```

Comment:
For now, this is hard-coded at 0. But we'll probably want to add some form of lotus API call to get the actual version given a height.

Reply:
This driver is actually not used any longer. We incubated it here, but we didn't erase it when it was moved to Lotus. This change needs to be applied here: https://github.com/filecoin-project/lotus/blob/master/conformance/driver.go#L125-L133. Will take care of it.

---

OTOH, I guess a hardcoded value is OK here for now, but we will want to change this in the test vector. How can we resolve, given a chain epoch, what network version to return here? There must be a function on Lotus side for that? (we don't need it exposed over the RPC).